centreon/www/class/centreon-clapi/centreonLDAP.class.php

```diff
@@ -171,89 +147,91 @@ public function show($params = array(), $filters = array())
     }
 
     /**
-     * @param null $arName
+     * @param string|null $arName
+     *
      * @throws CentreonClapiException
      */
-    public function showserver($arName = null)
+    public function showserver($arName = null): void
     {
-        if (is_null($arName) || !$arName) {
+        if (is_null($arName) || ! $arName) {
```

Comment:
useless `is_null`, thank you to remove it while modifing these lines.
```suggestion
        if (! $arName) {
```